Change fee token to gas token (#782)
OffchainLabs/arbitrum-docs: arbitrum-docs/launch-orbit-chain/orbit-quickstart.md

- A browser-based Ethereum wallet (like [MetaMask](https://chrome.google.com/webstore/detail/metamask/nkbihfbeogaeaoehlefnkodbefgpgknn))
- At least 1 testnet ETH (for custom gas token chains, 0.6 ETH and 0.4 native tokens)

## Step 1: Acquire Arbitrum Testnet $ETH (and the native token for Orbit chains with custom gas tokens)

> You'll need at least 1 testnet $ETH on the regular Orbit chains and 0.6 $ETH plus 0.4 of your desired native token for Custom Fee Token Orbit chains. The funds are needed to cover the cost of deploying your Orbit chain's **base contracts** to its **base chain** (Arbitrum Goerli or Sepolia). Sepolia is our recommendation as Goerli will be deprecated in the near future.

Author: ```suggestion
You'll need at least 1 testnet $ETH for regular Orbit chains or 0.6 $ETH plus 0.4 of your desired native token for Orbit chains with a custom gas token. The funds are needed to cover the cost of deploying your Orbit chain's **base contracts** to its **base chain** (Arbitrum Goerli or Sepolia). Sepolia is our recommendation as Goerli will be deprecated in the near future.
```
